src/dmd/backend/ty.d

```diff
@@ -183,6 +183,7 @@ enum
     mTYshared       = 0x00100000,    // shared data
     mTYnothrow      = 0x00200000,    // nothrow function
 
+    mTYreplaced     = 0x01000000,    // scalar replacement for aggregate
```

Comment:
Is the collision with the below flags intentional?

Reply:
Yes. Although is says in the comment "Used only by C/C++ compiler" I still can't find any of them in the backend. maybe there were used in some previous version of the backend.

Reply:
Hmm. @WalterBright ?

Reply:
Likely to be in the dmc repo if anywhere.